app/client/views/settings/users/form/manageUser.html

```diff
@@ -1,44 +1,64 @@
-<template name="editUser">
+<template name="manageUser">
   <div class="modal fade">
     <div class="modal-dialog">
       <div class="modal-content">
         <div class="modal-header">
           {{> flashMessages}}
           <h4 class="modal-title">
             <i class="fa fa-user"></i>&nbsp;
+            {{#if isEdit}}
             {{_ "editUser-header" }}
+            {{else}}
+            {{_ "newUser-header" }}
+            {{/ if}}
           </h4>
         </div>
-        {{# autoForm id="editUserForm" schema="EditUserSchema" doc=formUser }}
+        {{# autoForm id="userForm" schema="userSchema" doc=formUser }}
 
         <div class="modal-body">
           <label for="email">
             {{_ "users.email.label" }}
           </label>
           {{> afFieldInput name="email" }}
 
-          {{#if showIsAdminCheckBox}}
-          <label for="isAdmin">
-            {{_ "users.isAdmin.label" }}
+          <!-- Only show  this field in create mode-->
+          {{# unless isEdit}}
+          <label for="password">
+            {{_ "users.password.label" }}
           </label>
+          {{> afFieldInput name="password" }}
```

Comment:
By the way, lets make sure this is rendered as a `password` field, rather than a regular text field. I.e. it should not show the password as it is typed, but rather show a 'circle' for each character in the password.

Reply:
@brylie done.